lib/hello_world/auth/auth0_strategy.ex

```diff
@@ -0,0 +1,14 @@
+defmodule HelloWorld.Auth.Auth0Strategy do
+  @moduledoc """
+  Defines a custom Strategy for JokenJwks using a custom jwks domain.
+  """
+  use JokenJwks.DefaultStrategyTemplate
+
+  def init_opts(opts) do
+    Keyword.merge(opts, jwks_url: jwks_url())
+  end
+
+  defp jwks_url do
+    Application.get_env(:hello_world, :auth0_domain) <> "/.well-known/jwks.json"
```

Comment:
@manusajith I just realized that we are missing the HTTPS protocol here, that shouldn't come from the environment variable